app/views/hosts/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= javascript 'hosts', 'host_edit', 'host_edit_interfaces', 'class_edit', 'compute_resource', 'lookup_keys'%>
+<%= javascript 'hosts', 'host_edit', 'host_edit_interfaces', 'host_edit_puppetclasses', 'class_edit', 'compute_resource', 'lookup_keys'%>
```

Comment:
Though I don't think we have a way to trigger it right now, I'd add this JS to the hostgroup form for consistency - in case any errors also get logged against :puppetclasses there.